frontend/src/routes/candidate/(protected)/questions/+page.svelte

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+  import {page} from '$app/stores';
+  import {_} from 'svelte-i18n';
+  import Icon from '$lib/components/icon/Icon.svelte';
+  import {BasicPage} from '$lib/templates/basicPage';
+  import Button from '$lib/components/button/Button.svelte';
+
+  const firstQuestionUrl = `${$page.url.pathname.replace(/\/$/, '')}/${$page.data.questions[0].id}`;
```

Comment:
Does Svelte have a route builder we could use instead of relying on parsing `$page.url`? And`questions[0]` could be undefined